en/config/actuators.md

```diff
@@ -0,0 +1,116 @@
+# Actuator Configuration and Testing
+
+After the airframe is setup you should configure the geometry, assign actuators to outputs, and test the actuator response.
+This can be done in *QGroundControl*, under the **Vehicle Setup > Actuators** tab.
+
+:::note
+This view is only enabled if dynamic control allocation is enabled via [SYS_CTRL_ALLOC](../advanced_config/parameter_reference.md#SYS_CTRL_ALLOC), which will become the default in future.
+If you enable it, also make sure to select the airframe with [CA_AIRFRAME](../advanced_config/parameter_reference.md#CA_AIRFRAME).
+It replaces the existing mixer files and allows configuration via parameters instead.
+:::
+
+This is a screenshot for a multicotor:
+
+![Actuators MC (QGC)](../../assets/config/qgc_actuators_mc_aux.png)
+
+
+:::note
+If you find a setting missing (e.g. reversable motors), make sure to enable the **advanced** checkbox in the top right corner (not visible above).
```

Comment:
Sometimes worth "cheating" on the image. I.e. it is not possible to display the checkbox and the important parts of the image in the same view, so you take several shots and build a composite. Not sure in this case. Other option is to snapshot just the bit of interest.

Reply:
It was more lazyness of just using an existing screenshot. I updated it.